workflow/scripts/manhattan_plot.py

```diff
@@ -0,0 +1,45 @@
+import argparse
+import pandas as pd
+import matplotlib.pyplot as plt
+import subprocess
+import numpy as np
+
+def get_options():
+    description = 'Manhattan plot'
+    parser = argparse.ArgumentParser(description=description)
+    parser.add_argument('sample', help='Target mapped_all TSV file')
+    return parser.parse_args()
+
+def manhattan_plot(data):
+    df = pd.read_csv(data, sep='\t', usecols=['strain', 'start', 'end', 'lrt-pvalue', 'gene'])
+    df['pos'] = (df['start'] + df['end']) / (2 * 1_000_000)
+    reference_strains = df['strain'].unique()
+    t_output = subprocess.check_output(['python3','workflow/scripts/count_patterns.py','out/associations/phenotype/unitigs_patterns.txt'])
```

Comment:
Just a minor thing, but the `count_patterns.py` script has an argument called `--threshold` that only print the threshold, thus avoiding to have to parse the output